UNIFAQ compile errors

@JonWel your fix to the VTPR code broke compilation on a bunch of builders with errors like:
```
c:\users\ian\slave\csharp-windows\build\src\backends\cubics\UNIFAQ.h(23): error C2864: 'UNIFAQ::UNIFAQMixture::N' : only static const integral data members can be initialized within a class [C:\Users\ian\slave\Csharp-windows\build\build\CoolProp.vcxproj]
  c:\users\ian\slave\csharp-windows\build\src\backends\cubics\UNIFAQ.h(23): error C2864: 'UNIFAQ::UNIFAQMixture::N' : only static const integral data members can be initialized within a class [C:\Users\ian\slave\Csharp-windows\build\build\CoolProp.vcxproj]
  c:\users\ian\slave\csharp-windows\build\src\backends\cubics\UNIFAQ.h(23): error C2864: 'UNIFAQ::UNIFAQMixture::N' : only static const integral data members can be initialized within a class [C:\Users\ian\slave\Csharp-windows\build\build\CoolProp.vcxproj]
```
See also http://www.coolprop.dreamhosters.com:8010/grid.

Can you please fix this?

Message is now updated with UNIFAC.h (not UNIFAQ.h any-more):
```
  c:\users\ian\slave\csharp-windows32\build\src\backends\cubics\UNIFAC.h(23): error C2864: 'UNIFAC::UNIFACMixture::N' : only static const integral data members can be initialized within a class [C:\Users\ian\slave\Csharp-windows32\build\build\CoolProp.vcxproj]
  c:\users\ian\slave\csharp-windows32\build\src\backends\cubics\UNIFAC.h(23): error C2864: 'UNIFAC::UNIFACMixture::N' : only static const integral data members can be initialized within a class [C:\Users\ian\slave\Csharp-windows32\build\build\CoolProp.vcxproj]
  c:\users\ian\slave\csharp-windows32\build\src\backends\cubics\UNIFAC.h(23): error C2864: 'UNIFAC::UNIFACMixture::N' : only static const integral data members can be initialized within a class [C:\Users\ian\slave\Csharp-windows32\build\build\CoolProp.vcxproj]
```
I'll have a look on what it does not like.

--- a/src/Backends/Cubics/UNIFAC.cpp
+++ b/src/Backends/Cubics/UNIFAC.cpp
@@ -195,7 +195,9 @@ double UNIFAC::UNIFACMixture::ln_gamma_R(const double tau, std::size_t i, std::s
     }
 }
 void UNIFAC::UNIFACMixture::activity_coefficients(double tau, const std::vector<double> &z, std::vector<double> &gamma){
-    std::size_t N = z.size();
+    if (this->N != z.size()) {
+        throw CoolProp::ValueError("Size of molar fraction do not match number of components.");
+    }
     std::vector<double> r(N), q(N), l(N), phi(N), theta(N), ln_Gamma_C(N);
     double summerzr = 0, summerzq = 0, summerzl = 0;
     for (std::size_t i = 0; i < N; ++i) {

--- a/src/Backends/Cubics/UNIFAC.h
+++ b/src/Backends/Cubics/UNIFAC.h
@@ -20,7 +20,7 @@ namespace UNIFAC
     private:
         CoolProp::CachedElement _T; ///< The cached temperature
 
-        std::size_t N = 0; ///< Number of components
+        std::size_t N; ///< Number of components
 
         double m_T; ///< The temperature in K
         double T_r; ///< Reduce temperature

